cmd/program/program.go

```diff
@@ -711,6 +720,38 @@ func (p *Project) CreateHtmxTemplates() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	p.AdvancedTemplates.TemplateRoutes = template.HTML(routeBuffer.String())
-	p.AdvancedTemplates.TemplateImports = template.HTML(importBuffer.String())
+	p.AdvancedTemplates.TemplateRoutes = routeBuffer.String()
+	p.AdvancedTemplates.TemplateImports = importBuffer.String()
+}
+
+func (p *Project) CreateWebsocketImports(appDir string) {
+	websocketDependency := []string{"nhooyr.io/websocket"}
+	if p.ProjectType == flags.Fiber {
+		websocketDependency = []string{"github.com/gofiber/contrib/websocket"}
+	}
+
+	// Websockets require a different package depending on what framework is
+	// choosen. The application calls go mod tidy at the end so we don't
+	// have to here
+	err := utils.GoGetPackage(appDir, websocketDependency)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	importsPlaceHolder := ""
+	if p.AdvancedOptions["Websocket"] {
+		importsPlaceHolder += string(p.FrameworkMap[p.ProjectType].templater.WebsocketImports())
+	}
+
+	importTmpl, err := template.New("imports").Parse(importsPlaceHolder)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Nit: I want to fix the error handling in go-blueprint.
Can we do something like
```
if err != nil {
    log.Fatalf("function failed: %v", err)
}
```